tasks/xgettext.js

```diff
@@ -228,14 +242,15 @@ module.exports = function(grunt) {
         extractDirectiveStrings('"', func);
       });
 
-      return messages;
+      return [messages, messageToFilesMap];
     },
 
     handlebars: function(file, options) {
       var contents = grunt.file.read(file).replace("\n", " "),
```

Comment:
This replace is better done as a regular expression.

Reply:
how so?